lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -26,7 +26,7 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
-$src = Get-AnsibleParam -obj $params -name "src"
```

Comment:
Understand, now that the path type validates that the value is an actual path it would fail in the case it is a URL.